stl/src/xtime.cpp

```diff
@@ -8,10 +8,10 @@
 #include <time.h>
 #include <xtimec.h>
 
-#define NSEC_PER_SEC 1000000000L
+#define NSEC_PER_SEC  1000000000L
 #define NSEC_PER_MSEC 1000000L
 #define NSEC_PER_USEC 1000L
-#define MSEC_PER_SEC 1000

```

Comment:
not a change for this PR, but we could make these const or constexpr instead of macros (or just write them out, it's not like this will ever change).

Reply:
I will file an enhancement issue.

Reply:
Filed as #270.